src/Classes/Cli/HelpRenderer.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of MMLC - ModifiedModuleLoaderClient.
+ *
+ * (c) Robin Wieschendorf <mail@robinwieschendorf.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace RobinTheHood\ModifiedModuleLoaderClient\Cli;
+
+class HelpRenderer
+{
+    private const INDENT = '  ';
+    private string $description = '';
+    private array $arguments = array();
```

Comment:
In PHP you can use both array() and [] to create an array. Both spellings are valid and equivalent. However, the use of [] has become more common in recent PHP versions, especially from PHP 5.4 onwards, and is considered a more modern and shorter way of writing.